packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-reducer.ts

```diff
@@ -968,12 +978,12 @@ function boundAndSnapToPolygonAngle(
     });
 
     const getAngle = function (a: number, vertex, b: number) {
-        const angle = findAngle(
-            coordsCopy[rel(a)],
-            coordsCopy[rel(b)],
-            coordsCopy[rel(vertex)],
+        const allowReflexAngles = false; // Polygons do not have reflex angles
```

Comment:
I'm not sure this is right — polygons _can_ have reflex angles if they're concave. But it's not clear to me how this `getAngle` function is being used. I don't understand what `innerAngles` (below) is for. So this code might be doing the right thing.

What are the consequences of setting `allowReflexAngles` to true vs. false here?

Reply:
I can confirm that the polygon graph works identically to current prod, and that the reflexAngle setting does not appear to affect this either way. I've added a video to help illustrate! 

However there IS a pre-existing weird bug in the original reducer logic that might be worth a separate ticket. 🤔 It seems like we sometimes switch between inner and outer angles. @Myranae might know more about this. 

https://github.com/user-attachments/assets/b0c4bd1d-d465-47ee-a952-013cfa50a2b2

Reply:
Looks like the `angle-indicators.tsx` file might have some logic relating to the polygon angles; looks like you added this file in July, Third. I know there is logic  to prevent the shape from crossing over itself, but I don't _think_ that's relevant here.